openapi.yml

```diff
@@ -535,19 +535,33 @@ components:
     Offence:
       type: object
       properties:
-        date:
-          type: string
-          format: date
-          example: 1965-12-01
-          description: Date the offence took place or start date if the offence occurred over a period of time
-        code:
+        CJSCode:
           type: string
           example: RR84070
-          description: CJS offence code
+          description: Criminal Justice System offence code
+        courtDate:
+          type: string
+          format: date
+          example: 2018-02-10
+          description: Latest court date associated with the offence
         description:
           type: string
           example: Commit an act / series of acts with intent to pervert the course of public justice
           description: Description associated with the CJS offence code
+        endDate:
+          type: string
+          format: date
+          example: 2018-03-10
+          description: End date of range the offence was believed to have taken place
```

Comment:
"End date of range _over which_ the offence was believed to have taken place"

Reply:
Yes and I completely agree with you - the description that is there is pulled from the grammatically incorrect one from the Prison API